scripts/lochness_create_template.py

```diff
@@ -377,6 +410,14 @@ def get_arguments():
                         action='store_true',
                         help='Enable lochness to lochness transfer on the '
                              'sender side')
+    parser.add_argument('-rsync', '--rsync',
+                        default=False,
+                        action='store_true',
+                        help='Use rsync in lochness to lochness transfer')
+    parser.add_argument('-s3', '--s3',
```

Comment:
A script cannot have both `-s3` and `--s3`. For the latter, your README.md command runs into error.

Reply:
The same goes for `-rsync` and `--rsync`.

Reply:
To agree with [this](https://github.com/AMP-SCZ/lochness/tree/kcho/transfer_rsync#creating-the-template), we should just get rid of the short forms `-s3` and `-rsync`.

Reply:
https://github.com/AMP-SCZ/lochness/pull/36/commits/9e796cc7ee62c80ae5c2142841bd89197d827cec fixed